lib/generators/react_on_rails/templates/client/package.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Note, we can't put comments in the .json files (at least not conveniently).

Maybe we can place a nice README.md in the created /client directory that explains some things to customize.

In the below file, those would be:

1. node version
2. dependencies (documentation on how to update them here: https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/README.md)
3. difference between devDependencies and dependencies (anything needed for heroku deployment needs to go in "dependencies". Anything not needed for heroku deployment should go in "devDependencies".